library/general/volumeresources/samples/container-emptydir-limit/constraint.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: constraints.gatekeeper.sh/v1beta1
+kind: K8sVolumeRequests
+metadata:
+  name: container-emptydir-limit
+spec:
+  match:
+    kinds:
+      - apiGroups: [""]
+        kinds: ["Pod"]
```

Comment:
This policy will not work for pods since in rego, it is looking for `input.review.object.spec.template.spec` which is not available in `Pod`. I suggest writing this policy for `Pod` and then to impose same policy for workload resources `ExpansionTemplate` of gatekeeper could be used.

Reply:
Thanks @JaydipGabani 
Add `ExpansionTemplate` done.